src/ert/gui/tools/manage_experiments/storage_model.py

```diff
@@ -211,9 +221,30 @@ def headerData(
     def data(self, index: QModelIndex, role: int = Qt.ItemDataRole.DisplayRole) -> Any:
         if not index.isValid():
             return None
-
         return index.internalPointer().data(index, role)
 
+    @override
+    def flags(self, index: QModelIndex) -> Qt.ItemFlag:
```

Comment:
I suspect this should return multiple flags instead (the c++ function does that)
`virtual Qt::ItemFlags flags(const QModelIndex &index) const`

Reply:
Can you double check this? On first sight It does look like you're returning flags, and not flag?